crates/ty_python_semantic/resources/mdtest/import/star.md

```diff
@@ -1302,6 +1302,9 @@ scope of the name that was declared `global`, can add a symbol to the global nam
 
 ```py
 def f():
+    # TODO: Now that we lint on this `global` statement, maybe this whole test case should go?
```

Comment:
Yeah, I would remove this TODO, the one a few lines below it in `b.py`, and rework the prose above `a.py` to explain that we forbid the `global` statement in an inner scope if it refers to a symbol not declared or bound in the global scope, and that this is why we don't consider such symbols imported as a result of a `*` import in another module :-)

Reply:
7f2145e920f3dafcdd6691100cb28ceb32a5e703